crates/compiler/mono/src/ir.rs

```diff
@@ -4779,8 +4887,82 @@ pub fn with_hole<'a>(
             }
         }
 
-        TupleAccess { .. } => todo!(),
-        TupleAccessor(_) => todo!(),
+        TupleAccess {
```

Comment:
doesn't need to happen now but we should consolidate the logic of this branch with that of RecordAccess since they're largely the same.

Reply:
The here is subtly-different-enough that's it's really annoying to combine. I've already taken a few runs at it, unsuccessfully. I propose punting this for now.